src/components/TrendingTrack.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import styles from "./TrendingTrack.module.css";
+import playButton from "../images/play.svg";
+import likeButton from "../images/heart.svg";
+import moreButton from "../images/dots.svg";
+
+const TrendingTrack = ({ track }) => {
+  const [hovered, setHovered] = useState(false);
+
+  const handleHover = (e) => {
+    e.type === "mouseenter" ? setHovered(true) : setHovered(false);
+  };
+  return (
+    <>
+      <div className={styles.trackContainer}>
+        <div
+          className={styles.trackItem}
+          onMouseEnter={handleHover}
+          onMouseLeave={handleHover}
```

Comment:
I think it would be simpler to just be explicit here, and to remove the handleHover fn

```suggestion
          onMouseEnter={() => setHovered(true)}
          onMouseLeave={() => setHovered(false)}
```